src/components/modal/v2/parts/BodyContent.jsx

```diff
@@ -57,6 +57,7 @@ const BodyContent = () => {
 
     const { headline, subheadline, qualifyingSubheadline = '', closeButtonLabel } = content;
     const isQualifying = productMeta?.qualifying;
+    const useV4Design = productMeta?.useV4Design === 'true';
```

Comment:
```suggestion
    const useV4Design = productMeta?.useV4Design;
```
Instead of setting this to `true`, we can still set the variable, pass it into the `Header` further down in this file similar to how we are for `isQualifying`.

Reply:
Follow up question here - since` productMeta?.useV4Design` is a `string`, isn't it always going to always evaluate to `true` without the `=== 'true'` evaluation?

Reply:
Well, in addition to just having `const useV4Design = productMeta?.useV4Design;`we would have had to update where you are doing the ternary operators. I was just trying to keep things uniform with how we do things today, but you can actually revert back to include the `=== 'true'` evaluation since this is a temporary ramp and it'll keep things more simple to just do the check here.